Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -209,6 +209,9 @@ void handleSystemClock(void)
     if (sysClock >= INT_MAX)    sysClock = 0;
 
     UARTIntPrint("TS ", (int)sysClock);
+    //UARTIntPrint("RPM ", (int)readRotorRPM());
```

Comment:
Remove comments.

Reply:
Done, also added a card in trello regarding this.
https://trello.com/c/oZngu3oG/146-consider-creating-a-function-for-printing-stuff-through-uart